### ISSUE TITLE
Redesign hymn lists throughout the app

### ISSUE THREAD

**Issue by Emyeh** [HymnalDreamTeam/Hymns-iOS#208]
Hymn lists for Browse, categories, subcategories, search results... 

Android and iOS

**Comment by skywalkerdude** [comment]
@Emyeh I think we're somewhat blocked on development on this, since most of the other work is already done

**Comment by Emyeh** [comment]
What about something like this: https://www.figma.com/proto/7H5ApjzuVuJL3OP7FvE2qe/h-app?node-id=1369%3A1028&scaling=min-zoom

For lists I think it's better to keep it visually simple since it's not a main view. What's important here is providing context on what this list is and a clear view of the list items. I think the best way right now to make it more appealing is through animation. Lmk what you think.

**Comment by skywalkerdude** [comment]
Looks good! @Emyeh could I also get an error state for it? In case somehow the database call fails
